modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx

```diff
@@ -193,20 +193,23 @@ export function RightSidebarObjectDefinitionDetails({
 		}
 	};
 
+	const nodeDetailsTitle = sub(
```

Comment:
I think it would be better to rename it to `objectNodeDetailsTitle`, what do you think?